README.md

```diff
@@ -52,88 +44,251 @@ services.AddAuthentication(
         options.DefaultAuthenticateScheme = EasyAuthAuthenticationDefaults.AuthenticationScheme;
         options.DefaultChallengeScheme = EasyAuthAuthenticationDefaults.AuthenticationScheme;
     }
-).AddEasyAuth();
+).AddEasyAuth(this.Configuration);
 ```
 
-and this to the `public void Configure` method before `app.UseMvc...`:
+The next you must add the easy auth middleware to the ASP.NET pipeline. This can be enabled by the folling section in your `Configure`method in the `Startup`:
 
 ```csharp
 app.UseAuthentication();
 ```
 
-This will enable the `EasyAuthAuthenticationHandler` in your app.
+> Warning: make sure you add the `UseAuthentication` in the right order for your use case.
+
+The last thing is to add the following section in the `appsettings.json` to enabled the basic providers:
+
+```json
+"easyAuthOptions": {
+  "providerOptions": [
```

Comment:
Shouldn't be the line 
"AuthEndpoint": ".auth/me",
here as well?

Reply:
No @paule96 changed this in the last session. maybe the is a little catch with the implementation

Reply:
Hm, actually I get the web app not started, because it wants an .auth folder in wwwroot, which isn't there. This folder is only used for local debugging, when i have the correct understanding. In azure it is mapped by the surrounding web app, isn't it?

Reply:
There is no check if the folder is availed. Please look in to your looks. Maybe share the information with us. There are some new checks. 

We first test if you has configured you web app in the right way. If not it dies.

Then if a request is send to a protected endpoint and can't authorize it end the request, but the web app won't die

Reply:
When configuring easy auth like this: ArgumentNullException: Value cannot be null.<br />\r\nParameter name: The 'AuthEndpoint' in the configuraiton for easy auth can't be empty! please add the setting to your configuration providers.

Reply:
It is no problem to skip AuthEndpoint in Config, when using EasyAuthWithHeaderService. That's what I could see here with my WebApps in MCD.

Reply:
After a call with you, we maybe find the bug. Please share the result of your testing here so we can close this conversation.

Keep rocking. 🚀🧙‍♂️

Reply:
Now it looks good, authentication in my web apps works so far i can see. Thanks, Paule96! I have not tested, what happens if I come with an valid, but expired token to an app which uses EasyAuthForAuthorizationTokenService, but for normal authentification it works great.